### PR TITLE
CA 47

### PR DESCRIPTION
**Pull request by gengel**
* Removes dead-end upverters for 0.4.5 -> 0.4.6, 0.4.6 -> 0.4.7
* Adds internal version sequence doc
* Moves upverters into their own files, dynamically imported based on version sequence
* Adds command line interface
* Adds "create" command to generate schemas, upverter and test from current version
* Adds "cleanup" command to delete above
* getSchemas and getTransforms now pass "null" to callback on success (more idiomatic nodejs)
* package version de-coupled from ANS version

### CODE REVIEW

File: lib/util.js [hunk 1]
```diff
@@ -1,16 +1,6 @@
 'use strict';
```
Comment (rgfindl) [inline lib/util.js]: Is this file still needed?

Comment (gengel) [inline lib/util.js]: Nope, I'll remove it

**Comment by rgfindl** [comment]
Looks awesome.  Much cleaner.  Good work.  Merge when ready.
